### PR TITLE
NBO data files for subset of molecules

### PR DESCRIPTION
**Pull request by amandadumi**
Adding input and 2 output files related to NBO.

- The input files are the exact files from the ORCA5.0 data, the only addition being the NBO keyword added.
- the `.out` files have the Orca output with nbo embedded.
- the `.nbo.out` files have the isolated NBO output

I've only included a few to start, but would have to think about if any other methods make sense in terms of being chemically different enough that they could test another aspect of a parser.

Helps move #1012 forward.

### CODE REVIEW

File: data/NBO/nbo.out [hunk 1]
```diff
@@ -0,0 +1,10 @@
+./dvb_sp.inp
```
Comment (berquist) [inline data/NBO/nbo.out:1]: Any idea what this file is?

File: data/NBO/Trp_polar.inp [hunk 2]
```diff
@@ -0,0 +1,35 @@
+! hf sto-3g nori verytightscf noautostart NBO
```
Comment (berquist) [inline data/NBO/Trp_polar.inp:1]: Can we make this directory more specific?  Something like `data/NBO/basicNBO7.0_basicORCA5.0/...`?  (There's no precedence for this nesting AFAIK.)

File: data/NBO/Trp_polar.nbo.out [hunk 3]
```diff
@@ -0,0 +1,1032 @@
+
+ *********************************** NBO 7.0 ***********************************
+             N A T U R A L   A T O M I C   O R B I T A L   A N D
+          N A T U R A L   B O N D   O R B I T A L   A N A L Y S I S
+ *************************** Eric Berquist (I102531) ***************************
```
Comment (berquist) [inline data/NBO/Trp_polar.nbo.out:5]: Probably not important but good to keep in mind.  (See [Q27](https://nbo6.chem.wisc.edu/faq_css.htm).)  I think this usage is ok.

**Review by berquist — CHANGES_REQUESTED** [review]
The actual output looks fine as far as I can tell.  We do need to set the standard for naming directories of nested programs though.  (Also poking @shivupa for this.)

File: data/NBO/Trp_polar.inp (hunk 2 above)
Comment (amandadumi) [reply data/NBO/Trp_polar.inp]: I have introduced your suggested structure as it is better than just NBO top level, but certainly something we can discuss. The only other option would be to group them by two directories `basicNBOversion/basicCallingCodeversion/`

**Comment by berquist** [comment]
I like this idea much better.

File: data/NBO/Trp_polar.inp (hunk 2 above)
Comment (berquist) [reply data/NBO/Trp_polar.inp]: My email comment didn't work for some reason, I mean that this idea (`X/Y/`) is better than mine (`X_Y/`).

**Review by berquist — APPROVED** [review]
Thank you for doing all of this!

**Pull request merged**